### PR TITLE
vf: disable

### PR DESCRIPTION
**Pull request by cho-m**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

Only has a copyright https://github.com/glejeune/vf/blob/0.0.1/vf.rb#L337
```
    echo "vf v#{VF_VERSION}, (c) 2011 Gregoire Lejeune <gregoire.lejeune@free.fr>"
```

### CODE REVIEW

**Comment by Bo98** [comment]
https://docs.brew.sh/Deprecating-Disabling-and-Removing-Formulae indicates this should go straight to disabled.

File: Formula/v/vf.rb [hunk 1]
```diff
@@ -10,6 +10,8 @@ class Vf < Formula
     sha256 cellar: :any_skip_relocation, all: "e5e2d467c81269aec65859de0dd589b4493e9be0b77d047433ae1d2b98184bce"
   end
 
+  deprecate! date: "2024-08-06", because: "is missing a license"
```
Comment (SMillerDev) [inline Formula/v/vf.rb:13]:
```suggestion
  disable! date: "2024-08-06", because: "is missing a license"
```

**Review by carlocab — APPROVED** [review]

**Comment by Bo98** [comment]
Ok need to fix that infinite loop - that's not good.

**Comment by carlocab** [comment]
> Ok need to fix that infinite loop - that's not good.

It looks like one, but IIRC it should stop after five tries. (At least, if the check for five tries is correct. It may not be.)

**Pull request merged**